frontend/test/metabase/admin/tasks/containers/Logs.unit.spec.js

```diff
@@ -1,21 +1,19 @@
 import React from "react";
+import fetchMock from "fetch-mock";
 import { render, screen } from "@testing-library/react";
-import nock from "nock";
 import Logs from "metabase/admin/tasks/containers/Logs";
 
 import { UtilApi } from "metabase/services";
 
 describe("Logs", () => {
   afterEach(() => {
-    nock.cleanAll();
+    fetchMock.reset();
```

Comment:
Can we automate the same way we did for `nock` before?

Reply:
Yep. I wanted to do it in another PR because it might break things. E.g. I couldn't merge that PR with `nock`.